README.md

```diff
@@ -1 +1,24 @@
-# couch-tracker-documentation
\ No newline at end of file
+# Couch-Tracker
+
+An Android application, to keep track of the TV shows you follow, and the movies you've watched.
+
+This repository contains all the documents and design decisions we've made.
+
+
+## Tenets
+
+- Free and open-source
+- No ads, trackers, subscriptions, etc.
+- Maintaining and developing the app can be an hobby: low (or zero) fees, low architectural complexity, supported by the community
+- User data is owned by and hosted by the respective user
+- Fast and simple
+
+
+## General overview
+
+The app stores the user data in a file, logically owned by the user. The file will be in the user partition, and could be synced with other devices (e.g. using Synthing), backed up in the cloud, or read/written by other apps that understand the same file format.
+
+All the shows/movies/episodes/collections/actors referenced by the last open user data file will be downloaded and kept in sync automatically, this time in a local database owned by the app.
+This local data will be kept for a brief period of time after it is detected as unused, so switching between different user data files won't cause a complete re-download of everything.
+
+Notifications, widgets, and all other aspects of the app that don't require active interactions will reference the last open user data file.
```

Comment:
I don't like this "last open user data file" at all. I think we can either:
- explicitly make the app single db. There is no list of "recent" stuff open etc, simply the app assumes there is on db and one db only, which is what you set up on first install. If you go deep into the settings you can "logout" and start over with a new file, but I wouldn't make it easy if we don't support multi-db in a deeper way.
- have some sort of multi-db support where you can "add" dbs like you would add accounts on some other similar apps. This way the app can keep track of what is what and keep data/generate notifications and so on based on that.

In all honesty I think we can go with the first one, I don't see many use-cases where someone would need to have more than one db. On shared devices, Android already supports multiple accounts that gives each app its own data dir.

Reply:
Ok, let's go with option 1